scripts/functions.sh

```diff
@@ -26,14 +26,29 @@ function buildClient {
     cd ..
 }
 
+function doChecks {
```

Comment:
How about we move doChecks and go-bindata warnings into a separate pull request?
That's an unrelated but important feature

Reply:
okay, I gonna do it.